packages/workspace-tools/.eslintrc.json

```
@@ -0,0 +1,33 @@
+{
+  "extends": [
+    "../../.eslintrc.json"
+  ],
+  "ignorePatterns": [
+    "!**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts",
+        "*.tsx",
+        "*.js",
+        "*.jsx"
+      ],
+      "rules": {}
+    },
+    {
+      "files": [
+        "*.ts",
+        "*.tsx"
+      ],
+      "rules": {}
+    },
+    {
+      "files": [
+        "*.js",
+        "*.jsx"
+      ],
+      "rules": {}
+    }
+  ]
+}
```

```suggestion
{
  "extends": ["../../.eslintrc.json"]
}
```
Think we can simplify it as a result of my last week's update to `../../.eslintrc.json`.

---

Ah nice one, thanks! 👍